### PR TITLE
Total usage tile

### LINKED ISSUE #50: Implement the Average Usage dashboard tile
Implement the Total Usage dashboard tile

image

### PR DESCRIPTION
**Pull request by benoit74**
## Rationale

Fix #50

## Changes
- Implement the Total Usage Dashboard Tile, including creation of a custom store with getters dedicated to this tile

### CODE REVIEW

**Comment by benoit74** [comment]
Review #60 first and then rebase this.

File: frontend/src/utils.ts [hunk 1]
```diff
@@ -23,3 +23,12 @@ export const hexToRgb = (hex: string) => {
       )}`
     : null
 }
+
```
Comment (rgaudin) [inline frontend/src/utils.ts:26]: Output format should be explicit

**Review by rgaudin — APPROVED** [review]
LGTM (albeit the missing docstring about expected color format)

File: frontend/src/utils.ts (hunk 1 above)
Comment (benoit74) [reply frontend/src/utils.ts:26]: Explained in comment

**Pull request merged**